test/Scrabble/Move/MoveTests.hs

```diff
@@ -0,0 +1,78 @@
+{-# LANGUAGE TemplateHaskell #-}
+
+module Scrabble.Move.MoveTests where
+
+import Test.Framework.Providers.HUnit
+import Test.Framework.TH
+import Test.HUnit
+import TestHelpers
+import Scrabble.Move.Move
+import Scrabble.Board.Board
+import Scrabble.Bag
+
+
+case_create_move =
+    let rack = Rack    [ fromLetter O, fromLetter P, fromLetter T ]
+        wp   = WordPut [ LetterTilePut (fromLetter T) (7,7)
```

Comment:
`WordPut.makeWordPut`. I think that should help make this code shorter. What do you think?